.github/workflows/eleventy_build.yml

```diff
@@ -15,14 +15,14 @@ jobs:
         - name: Setup Node
           uses: actions/setup-node@v1
           with:
-            node-version: '10.x'
```

Comment:
Didn't realize this PR included the .yml change -- I just merged this change and it fixed the problem, so I'm closing this PR.